### PR TITLE
Add information about RMA shipping methods

### PR DESCRIPTION
**Pull request by ZhovnirMaria**
## Purpose of this pull request

_Describe the goal and the type of changes this pull request covers. Tell us what changes you are making and why._

This pull request:
- adds information about RMA shipping methods;
- adds commerce label to the Autosetting page

## Magento release version

_Which Magento release(s) are affected by the content changes: 2.4, 2.3? Specify a patch release number, if applicable._

## Product editions

_Is this update specific to a product edition: Magento Open Source only, Adobe Commerce only?_

- yes

_Is this update specific to an installed feature extension: B2B extension, other feature set (please, specify)?_

- no

## Affected documentation pages

https://docs.magento.com/user-guide/sales/rma-configure.html
https://docs.magento.com/user-guide/catalog/product-autosettings.html?itm_source=merchdocs&itm_medium=quick_search&itm_campaign=federated_search&itm_term=auto

### CODE REVIEW

File: src/catalog/product-autosettings.md [hunk 1]
```diff
@@ -9,3 +9,5 @@ The Autosettings section includes any attributes that are dependencies for other
 
 ![]({% link images/images-ee/product-autosettings.png %}){: .zoom}
 _Autosettings_
+
+With RMA, the admin has the ability to allow return functionality for specific products, and vice versa could also disallow the returning of specific products.
```
Comment (leslietilling) [inline src/catalog/product-autosettings.md:13]:
```suggestion
With RMA, the an Admin user has the ability to allow return functionality for specific products, as well as disallow the returning of specific products.
```

**Review by leslietilling — CHANGES_REQUESTED** [review]
Minor edits for clarity and consistency

File: src/catalog/product-autosettings.md (hunk 1 above)
Comment (ZhovnirMaria) [reply src/catalog/product-autosettings.md]: Done, thanks 🙏

**Review by leslietilling — APPROVED** [review]

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@leslietilling the pull request successfully imported.

**Pull request merged**